docs/source/development/automated_testing.md

```diff
@@ -19,6 +19,17 @@ There are many testing frameworks available for Python. One of the most popular
 
 Let's look at how you can start working with `pytest` in your Kedro project.
 
+### Prerequisite: Install your Kedro project
+
+Before getting started with `pytest`, it is important to ensure you have installed your project locally. This allows you to test different parts of your project by importing them into your test files.
+
+To install your project, navigate to your project root and run the following command:
+
+```bash
+pip install -e .
+```
+
+>**NOTE**: The option `-e` installs an editable version of your project, allowing you to make changes to the project files without needing to re-install them each time.
```

Comment:
Maybe let's use native MyST syntax

````suggestion
:::{note}
The option `-e` installs an editable version of your project, allowing you to make changes to the project files without needing to re-install them each time.
:::
````

https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html

(fenced admonitions should also work)

Reply:
The MyST syntax doesn't seem to render properly on RTD. Additionally, the syntax of `>**NOTE**` is currently consistent across the docs - I'm unsure of the motivation to change it

Reply:
True, we needed the `colon_fence` extension